src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,48 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  constructor(name, health) {
+    this.name = name;
+    this.health = 100;
```

Comment:
The `health` parameter in the constructor is not being used. Instead, the health is always set to 100. Consider using the `health` parameter to initialize the `health` property.